src/plugins/vis_type_timeseries/server/lib/get_fields.ts

```diff
@@ -62,7 +62,7 @@ export async function getFields(
   if (!indexPatternString) {
     const defaultIndexPattern = await indexPatternsService.getDefault();
 
-    indexPatternString = defaultIndexPattern?.title ?? '';
+    indexPatternString = defaultIndexPattern?.patternListActive.join(',') ?? '';
```

Comment:
I'm not a big fan to do `patternListActive.join(',')` on each places where we need to get string representation of index pattern. From my perspective it should be encapsulated as a method in IndexPattern service: e.g `getIndexPatternString()` or just a `toString()`